FORMAT.md

```diff
@@ -71,15 +76,25 @@ The actuator element represents actuators such as the X5-4.  It is assumed to ha
 
 The link element refers to a parameterized rigid body with two parameters (extension and twist).  All links have one output interface.
 
+Note that the "extension" and "twist" values generally correspond to those shown on http://docs.hebi.us/hardware.html.
```

Comment:
Is there a reason for the word "generally"? Overall things look good to me.

Reply:
Mostly because we don't have all of the links up on docs.hebi.us, so this isn't documented for the new links.  I can change with the idea that this will be updated soon.